app/src/ui/branches/branch-list.tsx

```diff
@@ -239,7 +239,7 @@ export class BranchList extends React.Component<
     if (this.props.canCreateNewBranch) {
       return (
         <Button className="new-branch-button" onClick={this.onCreateNewBranch}>
-          New
+          New Branch
```

Comment:
Can we make this `{__DARWIN__ ? 'New Branch' : 'New branch'}` pls?

Reply:
Sure can. Done in 2ff48c9